diff --git a/.prettierignore b/.prettierignore
new file mode 100644
index 0000000000000000000000000000000000000000..fa29cdfff915566d202c195607a42b9445919261
--- /dev/null
+++ b/.prettierignore
@@ -0,0 +1 @@
+**
\ No newline at end of file
diff --git a/dist/client/components/app-router.js b/dist/client/components/app-router.js
index aa2818d8ead9ed713077ff715a411d04dc149ee5..b15f914df85079a31870b0a09279be08019e503f 100644
--- a/dist/client/components/app-router.js
+++ b/dist/client/components/app-router.js
@@ -376,6 +376,7 @@ function Head(param) {
         const originalReplaceState = window.history.replaceState.bind(window.history);
         // Ensure the canonical URL in the Next.js Router is updated when the URL is changed so that `usePathname` and `useSearchParams` hold the pushed values.
         const applyUrlFromHistoryPushReplace = (url)=>{
+            if (!window.history.state) return;
             const href = window.location.href;
             (0, _react.startTransition)(()=>{
                 dispatch({
